_docs/10-download-releases.md

```diff
@@ -11,10 +11,26 @@ The table below shows the supported jre environment for components of each relea
 | 4.0.0-incubating | >=1.7 | >=1.8 | >=1.8 |
 | 4.1.0-incubating | >=1.6 | >=1.8 | >=1.8 |
 | 4.2.0 | >=1.6 | >=1.8 | >=1.8 |
-
+| 4.3.x | >=1.6 | >=1.8 | >=1.8 |
 
 # Download links
-Use the links below to download the Apache RocketMQ Releases, the latest release is 4.2.0.
+Use the links below to download the Apache RocketMQ Releases, the latest release is 4.3.2.
```

Comment:
It is recommend to link these download address to the release note instead of copying them again.